### ISSUE TITLE
[accessibility] The slider control toggle is not keyboard operable

### ISSUE THREAD

**Issue by Malvoz** [Eclipse1979/leaflet-slider#6]
The `<a>` slider control is not keyboard operable. Additionally, it is announced as a "link" using a screen reader (missing [`role="button"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role), though [`<button type="button">` is recommended](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/button/button.html)).

Also see <span title="Web Accessibility Initiative - Accessible Rich Internet Applications">WAI-ARIA</span> Authoring Practices [recommended keyboard interactions](https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-3) for [buttons](https://www.w3.org/TR/wai-aria-practices/#button).

**Comment by Malvoz** [comment]
Other potentially helpful resources:
- [Leaflet Guideline: Test your maps](https://leafletjs.com/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
